frontend/src/component/FeedItem/FeedItemHeader.jsx

```diff
@@ -148,14 +157,29 @@ export class FeedItemHeader extends React.Component {
               <i className={`fa-fw ${props.contentType.faIcon}`} />
               {props.t('Open as content')}
             </Link>
+
+            {shouldShowChangeTypeButton && (
+              <IconButton
+                customClass='feedItemHeader__actionMenu__item'
+                disabled={props.content.is_archived || props.content.is_deleted}
+                icon='fa-fw fas fa-fw fa-th'
```

Comment:
no need for fa-fw twice

Reply:
also, can you explain the choice for fa-th?

Reply:
it's to have the same icon as the content tab

Reply:
I see, it's a good idea. Though I wonder whether it should be the icon of app file since we can only turn contents into content type file.

My point is that the icon for contents doesn't appear a lot in tracim and users might be more used to the icon of content type file which appear more often.

I think you can ask that to the team to get some insights

Reply:
i change for fa-comments witch is the icon for discussion

Reply:
The content didn't became file type content, they became discussion, and for now the only possible change are between publication and thread

Reply:
agreed